sqlglot/optimizer/qualify_columns.py

```diff
@@ -202,10 +202,21 @@ def _get_selectable_columns(name, selectables, schema):
 
     # If referencing a table, return the columns from the schema
     if isinstance(selectable, exp.Table):
+        catalog_name = selectable.text("catalog")
```

Comment:
can we make it so catalog and db are optional? so gracefully handle the cases where the schema object has different levels of nesting? i presume in many cases, we'd only pass in tables, and in most cases only db/table in schema. i don't think we'll ever use catalog